### PR TITLE
add new workflows

### PR DESCRIPTION
**Pull request by nbulsara**
*Description of changes:*
Added new workflows scrnaseq (with cellranger) , fastqc, vep, taxprofiler, hisat-genotype 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by markjschreiber** [comment]
Zoiks!! 848 Files Changed!! Did these come from the hackathon?

**Comment by nbulsara** [comment]
> Zoiks!! 848 Files Changed!! Did these come from the hackathon?

Yes!

**Comment by wleepang** [comment]
I will presume that the 848 file change is largely adding new files (workflow definitions). Will triage review to changed pre-existing files.

File: example-workflows/nf-core/workflows/fastqc/README.md
Comment (wleepang) [inline example-workflows/nf-core/workflows/fastqc/README.md:1]: Add a banner to the top that this is a point in time copy of the nf-core/fastqc workflow

File: example-workflows/nf-core/workflows/taxprofiler/container_substitutions.json
Comment (wleepang) [inline example-workflows/nf-core/workflows/taxprofiler/container_substitutions.json:1]: Is this file needed since ecr-helper can handle retrieving images from Dockerhub?

File: example-workflows/other_WDL/README.md
Comment (wleepang) [inline example-workflows/other_WDL/README.md:1]: Is this the right README file for this workflow grouping?

File: example-workflows/other_nextflow/workflows/vep/README.md
Comment (wleepang) [inline example-workflows/other_nextflow/workflows/vep/README.md:1]: Add a notification that this workflow is a point in time snapshot

**Review by wleepang — COMMENTED** [review]
848 files - Whew!
Overall, looks good. Just a couple comments.

File: example-workflows/nf-core/workflows/fastqc/README.md
Comment (nbulsara) [reply example-workflows/nf-core/workflows/fastqc/README.md:1]: this is a custom workflow built using the fastqc and multiqc modules and they're not tracked with versions similar to workflows, ive added a note though

File: example-workflows/nf-core/workflows/taxprofiler/container_substitutions.json
Comment (nbulsara) [reply example-workflows/nf-core/workflows/taxprofiler/container_substitutions.json:1]: not needed, deleted it.

File: example-workflows/other_WDL/README.md
Comment (nbulsara) [reply example-workflows/other_WDL/README.md:1]: yes this is similar to the protein folding grouping of workflows

File: example-workflows/other_nextflow/workflows/vep/README.md
Comment (nbulsara) [reply example-workflows/other_nextflow/workflows/vep/README.md:1]: similar to fastqc and multiqc, this is a module not tracked by versions. added a warning in the README though

Comment (wleepang) [inline example-workflows/other_nextflow/workflows/vep/README.md:1]: Add a statement that this workflow is a point in time snapshot of upstream

File: example-workflows/other_WDL/README.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# AWS HealthOmics Protein Workflows
```
Comment (wleepang) [inline example-workflows/other_WDL/README.md]: This README needs to update references to protein folding to ones indicative of the workflows in the grouping - i.e. HLA

**Review by wleepang — COMMENTED** [review]
just a couple more comments

File: example-workflows/other_WDL/README.md (hunk 1 above)
Comment (nbulsara) [reply example-workflows/other_WDL/README.md]: yes this change was pushed last night.

File: example-workflows/other_nextflow/workflows/vep/README.md
Comment (nbulsara) [reply example-workflows/other_nextflow/workflows/vep/README.md:1]: since it was an nfcore/module which doesnt maintain versioning it wont make sense to mention that

File: example-workflows/other_WDL/README.md
Comment (wleepang) [inline example-workflows/other_WDL/README.md:1]: lines 128-158 need updates too

Comment (nbulsara) [reply example-workflows/other_WDL/README.md:1]: done

**Review by wleepang — APPROVED** [review]
